client/src/search/Typeahead/Typeahead.scss

```diff
@@ -1,124 +1,110 @@
+          height: 0;
+        }
     }
 }
-
-
-//The following classes are taken from react-bootstrap-typeahead css
-
-.rbt {
-  outline: none;
-}
-
-.rbt-highlight-text {
-  background-color: inherit;
-  color: inherit;
-  font-weight: bold;
-  padding: 0;
+.typeahead__icon-container {
+  display: flex; 
+  align-items: center;
 }
 
-.rbt-menu {
-  margin-bottom: 2px;
+.typeahead__dropdown {
+  position: absolute;
+  z-index: 1000;
   width: 100%;
+  max-width: 100vw;
+  max-height: 400px;
+  overflow-y: auto;
+  margin-bottom: 2px;
   margin-top: 12px;
   padding: 8px 0;
   background-color: $backgroundColor;
   border: 1px solid $separatorColor;
   border: 1px solid rgba(0, 0, 0, 0.2);
   border-radius: $standardBorderRadius;
   box-shadow: 0 5px 10px rgba(0,0,0,.2);
-  max-height: 400px;
-  overflow-y: auto;
-  max-width: 100vw;
-  // list-style-type: none;
-
-  & > .dropdown-header, & > div > .dropdown-header {
-    margin-top: 0px;
-    padding: 0px 20px 3px 10px;
-    font-size: 1.8em;
-    line-height: 1.8em;
-    white-space: normal;
-    border-bottom: 1px solid $separatorColor;
-    &:not(:first-child) {
-      border-top: 1px solid $separatorColor;
-    }
+}
 
-    &:hover {
-      background-color: $backgroundColor;
-      color: #777;
-    }
+.typeahead__header {
+  margin-top: 0px;
+  padding: 0px 20px 3px 10px;
+  font-size: 1.8em;
+  line-height: 1.8em;
+  white-space: normal;
+  border-bottom: 1px solid $separatorColor;
+  color: #777;
+
+  &:not(:first-child) {
+    border-top: 1px solid $separatorColor;
   }
+}
 
-  & > li {	
-    padding: 3px 20px;	
-    font-size: 1.4em;	
-    line-height: 1.8em;	
-    cursor: pointer;	
-
-    &:not(:first-child) {	
-      border-top: 1px solid $separatorColor;	
-    }	
-
-    & > a {	
-      white-space: normal;	
-      color: $linkColor;	
-    }	
+.typeahead__item {	
+  padding: 3px 20px;	
+  font-size: 1.4em;	
+  line-height: 1.8em;	
+  cursor: pointer;	
+
+  &:not(:first-child) {	
+    border-top: 1px solid $separatorColor;	
+  }	
+
+  &--active,
+  &--active,
+  &:focus,
+  &:hover {
+      background-color: lighten($tertiaryColor, 30%);
+      color: $linkFocusColor;
+      text-decoration: underline;
   }
 }
-.rbt-menu > li a {
+
+.typeahead__result,
+.typeahead__control {
+  display: block;
+  padding: 3px 20px;
   overflow: hidden;
   text-overflow: ellipsis;
+  white-space: normal;	
+  color: $linkColor;
 }
 
-.rbt-menu > li a:focus {
-  outline: none;
-}
-
-.rbt-menu-center {
-  text-align: center !important;
-}
-
-.rbt-menu.dropdown-menu > li.active,
-.rbt-menu.dropdown-menu > li.active > a,
-.rbt-menu.dropdown-menu > li:focus:not(.dropdown-header),
-.rbt-menu.dropdown-menu > li:hover:not(.dropdown-header),
-.rbt-menu.dropdown-menu > li > a:focus,
-.rbt-menu.dropdown-menu > li > a:hover {
-  &:not(.disabled){
-    background-color: lighten($tertiaryColor, 30%);
-    color: $linkFocusColor;
-    text-decoration: underline;
-  }
+.typeahead__control {
+  text-align: center;
```

Comment:
Is there a reason for this to be separated?

Reply:
I figured they might diverge more at some point. I also wanted to keep the terminology around the menu clear, because we've got both controls and results side-by-side in it (and awkwardly sharing things like the selection cursor).